src/ios/PurchasesPlugin.swift

```diff
@@ -373,15 +373,15 @@ public class CDVPurchasesPlugin : CDVPlugin {
 
 extension CDVPurchasesPlugin: PurchasesDelegate {
 
-    private func purchases(_ purchases: Purchases, didReceiveUpdated purchaserInfo: CustomerInfo) {
-        let result = CDVPluginResult(status: .ok, messageAs: purchaserInfo.dictionary)
+    public func purchases(_ purchases: Purchases, receivedUpdated customerInfo: CustomerInfo) {
```

Comment:
@aboedo turns out we can't have this private in Swift 5 😆

Reply:
ohhh that makes sense. That's why I asked the previous time, I vaguely recalled Xcode yelling at me when trying to have private implementations of public delegate methods